ui/src/app/views/workspace-edit/component.html

```diff
@@ -85,8 +86,9 @@
       </div>
     </section>
   </div>
-  <div class="button">
+  <div style="display: inline-block" class="button">
     <button class="btn add-button btn-primary" [clrLoading]="buttonClicked" (click)="addWorkspace()">Create Workspace</button>
     <button class="btn" routerLink="/../..">Cancel</button>
   </div>
+  <div *ngIf="this.valueNotEntered;" style="display: inline-block">* is a required field</div>
```

Comment:
This should be a `<p>` since it's just containing text. I think that may get you the `display` you want for free.

Reply:
I changed it to `<p>` but that doesn't actually get the display for free. (I agree that it should be a paragraph tag though)